index.js

```diff
@@ -1,17 +1,43 @@
+var path = require('path')
 var fs = require('fs-extra')
+var minimatch = require('minimatch')
+
+function performMatch (entry, globs) {
```

Comment:
should we use: https://www.npmjs.com/package/matcher-collection for this? As it is has tests: https://github.com/stefanpenner/matcher-collection/blob/master/test.js

Reply:
I think so

Reply:
sounds good. I'll good with it either way. Your call

Reply:
updated to use matcher-collection